APMrover2/defines.h

```diff
@@ -27,13 +27,31 @@ enum ch7_option {
     CH7_RTL             = 9,
     CH7_SMART_RTL       = 10,
     CH7_GUIDED          = 11,
-    CH7_LOITER          = 12
+    CH7_LOITER          = 12,
+    CH7_FOLLOW          = 13
 };
 
 // HIL enumerations
 #define HIL_MODE_DISABLED 0
 #define HIL_MODE_SENSORS  1
 
+
+// Auto Pilot modes
+// ----------------
+enum mode {
```

Comment:
I think we've moved this module enum into mode.h so it should be added there instead of here.
Also we can't changed the number for existing modes.  So for example, pushing INITIALISING can't be moved from 16 to 17.  As a side note, I would actualy make use of one of the blank slots in the enum.  For example make Follow "6".  I'm not sure why we have those blank spaces, there's no need for them.